test/e2e/crd_e2e_test.go

```diff
@@ -304,8 +307,21 @@ var _ = Describe("CRD Versions", func() {
 
 		// update channel on sub
 		sub.Spec.Channel = alphaChannel
-		_, err = crc.OperatorsV1alpha1().Subscriptions(testNamespace).Update(context.TODO(), sub, metav1.UpdateOptions{})
-		Expect(err).ToNot(HaveOccurred(), "could not update subscription")
+		sub.TypeMeta.APIVersion = "operators.coreos.com/v1alpha1"
+		sub.TypeMeta.Kind = "Subscription"
+		sub.ManagedFields = []metav1.ManagedFieldsEntry{}
+
+		// use server-side apply to apply the update to the subscription point to the alpha channel
+		forceApply := true
+		opts := metav1.PatchOptions{
+			Force:        &forceApply,
+			FieldManager: "tests",
+		}
+		newSub, err := client.Apply.Data(sub)
+		Expect(err).ToNot(HaveOccurred(), "could not serialize updated subscription to bytes")
+
+		_, err = crc.OperatorsV1alpha1().Subscriptions(testNamespace).Patch(context.TODO(), sub.GetName(), types.ApplyPatchType, newSub, opts)
```

Comment:
@exdx I built a [test utility](https://github.com/operator-framework/operator-lifecycle-manager/blob/c3bcd2eaacd0b6ea326b0a521f97d10b8d18d483/test/e2e/util_test.go#L916) that handles server-side apply generically, sans boilerplate. Is there a reason we shouldn't/can't use that here too?

Ex.

```go
Eventually(Apply(sub, func(s *operatorsv1alpha1.Subscription) error {
    s.Spec.Channel = alphaChannel
    s.TypeMeta.APIVersion = "operators.coreos.com/v1alpha1"
    s.TypeMeta.Kind = "Subscription"
    s.ManagedFields = []metav1.ManagedFieldsEntry{}
})).Should(Succeed())
```

Reply:
Should this be returning an error somehow?

Reply:
typo, sorry